packages/storefront-automation/tests/home_test.js

```diff
@@ -0,0 +1,13 @@
+Feature('home');
```

Comment:
Since the test is under "tests" directory, I suggest we name this file as "home.js" instead of "home_test.js"